analytics/math3/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,12 @@
+
+Apache Edgent: Analytics: Math3
+Copyright 2017 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+===============================================================================
+
+Portions of this bundle were developed by IBM Corp.
+Copyright IBM Corp. 2015, 2016
+
```

Comment:
Yikes... that's pretty cumbersome :-)  
"Copyright 2017 The Apache..." needs to be "Copyright 2016-2017...", right?
And we'll need to remember to change them all the first release of a year.

Reply:
Another issue? ...the original binary bundle jar NOTICE (in licenses/ibm-contrib-notice) includes an incubating disclaimer sentence. It's omitted in these.  I suspect it's required.

Reply:
Well the option I would probably go for is to make a resource bundle like the one used in the remote resources bundle. The year could be automatically inserted by maven ... in the worst case it would be just a property on the parent project. This is just the quick fix for the open issue. I used the content of the official template the apache foundation provides for incubating projects. The incubating message is in the Disclaimer and not in the Notice for these. If this were required, probably the Apache Foundation would have updated those templates. And as Justin said "Keep it as short as possible" :-)

Reply:
I didn't realize it wasn't present in the official template... excellent!

Reply:
oh... but we still a starting date of 2016 before we can release this